packs/email/rules/sample.emailsample.yaml

```diff
@@ -0,0 +1,25 @@
+---
+uid: 'rule:email:Sample'
```

Comment:
uid is actually optional (it's automatically inferred from other fields), you can remove it from the resource metadata file.